### PR TITLE
Added logcli docker image

### LINKED ISSUE #1751: Provide logcli docker image
**Is your feature request related to a problem? Please describe.**

I'd like to install logcli as a docker container, because installing binaries on some OS  (i.e.  k3os/coreos) is discouraged in favor of container installations.
However. the official [grafana/logcli](https://hub.docker.com/r/grafana/logcli/tags) is lacking tags and is therefore not usabale. I also don't like to install untrusted images from docker hub.

**Describe the solution you'd like**

Please publish latest logcli docker images, or include logcli inside the `grafana/loki` image.

**Describe alternatives you've considered**

* Maintaining your own image, which comes with extra overhead for a lot of users
* Installing the binary as descibed in the docs.

Thanks !

### PR DESCRIPTION
**Pull request by tharun208**
**What this PR does / why we need it**:
    This PR introduces the docker image for logcli
**Which issue(s) this PR fixes**:
Fixes #1751 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: cmd/logcli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.13 as build
```
Comment (slim-bean) [inline cmd/logcli/Dockerfile:1]: could you change this to match the go version in the other docker files? 1.14.2?

**Comment by slim-bean** [comment]
Hey @tharun208 thanks for the PR!

For the drone generation, we use jsonnet to define the jobs and then auto-generate the yaml using `make drone`

Could you look at updating drone.jsonnet (I think hopefully it's as easy as adding logcli to the list of apps at the top) and then run `make drone` to generate the drone.yaml file?

File: cmd/logcli/Dockerfile (hunk 1 above)
Comment (tharun208) [reply cmd/logcli/Dockerfile:1]: sure

**Comment by tharun208** [comment]
> Hey @tharun208 thanks for the PR!
> 
> For the drone generation, we use jsonnet to define the jobs and then auto-generate the yaml using `make drone`
> 
> Could you look at updating drone.jsonnet (I think hopefully it's as easy as adding logcli to the list of apps at the top) and then run `make drone` to generate the drone.yaml file?

@slim-bean regenerated the `drone.yml` and updated `drone.jsonnet` 👍

**Pull request merged**